test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -2531,79 +2535,145 @@ class test_ucp_sockaddr_protocols_err_sender
                                                "rcache,odp,direct"));
     }
 
-    ucs::ptr_vector<ucs::scoped_setenv> m_env;
-};
+    /* This test is quite tricky: it checks for incorrect behavior on RNDV send
+     * on DC transport: in case if sender EP was killed right after sent RTS
```

Comment:
maybe mention that it tries to find the one which supports `UCT_MD_FLAG_INVALIDATE`. Now it is DC, but maky take more transports when invalidate support added to them

Reply:
done